modules/account-lib/src/coin/near/utils.ts

```diff
@@ -46,16 +46,12 @@ export class Utils implements BaseUtils {
    * or can create a new keyPair
    */
   isValidAccountId(accountId: string): boolean {
-    try {
-      // this regex is from near doc, https://docs.near.org/docs/concepts/account
+    return (
       (/^(([a-z\d]+[\-_])*[a-z\d]+\.)*([a-z\d]+[\-_])*[a-z\d]+$/.test(accountId) &&
         accountId.length > 2 &&
```

Comment:
@facundoCarreSimtlix According to the near [doc](https://docs.near.org/docs/concepts/account), 2 is a valid account length so shouldn't it be `accountId.length >= 2`?

Reply:
yes, You are right, i fix that